plugin-core/src/main/java/appland/actions/PluginStatus.java

```diff
@@ -122,8 +122,12 @@ private static String appmapBinaryStatusReport(@NotNull CliTool type) {
         var downloadPath = CliTools.getBinaryPath(type);
 
         String localVersionText = "Your version: ";
-        localVersionText += downloadedVersion == null ? "<unavailable>" : downloadedVersion;
-        localVersionText += downloadPath == null ? "<unavailable>" : String.format("\n\nLocation: %s", downloadPath);
+        if (downloadedVersion == null || downloadPath == null) {
+            localVersionText += "&lt;unavailable>";
```

Comment:
Is the HTML entity here intentional? I don't think they're required (or if they're even supposed to be supported) in markdown.